app/src/androidTest/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreenTest.kt

```diff
@@ -72,11 +68,95 @@ class SplashScreenTest : BaseAndroidComposeTest() {
         }
     }
 
+    @Test
+    fun when_type_email_into_email_text_field_has_email_text() {
+        setupSplashComposeRule()
+        with(composeTestRule) {
+            val email = "example@gmail.com"
+            onNodeWithContentDescription(getString(R.string.login_email_text_field))
+                .performTextInput(email)
+            onNodeWithContentDescription(getString(R.string.login_email_text_field))
+                .assert(hasText(email))
+        }
+    }
+
+    @Test
+    fun when_type_password_into_password_text_field_has_text_with_mask() {
+        setupSplashComposeRule()
+        with(composeTestRule) {
+            val password = "p"
+            onNodeWithContentDescription(getString(R.string.login_password_text_field))
+                .performTextInput(password)
+            onNodeWithContentDescription(getString(R.string.login_password_text_field))
+                .assert(hasText("\u2022"))
+        }
+    }
+
+    @Test
+    fun when_fill_in_only_one_field_login_button_is_disabled() {
+        setupSplashComposeRule()
+        with(composeTestRule) {
+            val email = "example@gmail.com"
+            onNodeWithContentDescription(getString(R.string.login_email_text_field))
+                .performTextInput(email)
```

Comment:
Same here since it's used only once.
```suggestion
                .performTextInput("example@gmail.com")
```